src/snapshotter.test.js

```diff
@@ -124,4 +125,19 @@ describe('The snapshotter', () => {
     await mockSnapshot.takeSnap();
     expect(mockSnapshot.driver.addCookie.mock.calls.length).toBe(2);
   });
+
+  it('Executes a script', async () => {
+    const config = {
+      gridUrl: 'https://lol.com',
+      url: 'http://cps-render-ci.elb.tnl-dev.ntch.co.uk/',
+      label: '1homepage',
+      onBeforeScript: './src/__mocks__/seleniumMock.js',
```

Comment:
@shendriksen this is something I thought up, hope you like it. Part of me thinks it smells, part of me thinks 💪

Reply:
Looks good. I think we should have a generic url as it is a public repo though.